src/addrman.h

```diff
@@ -376,8 +396,17 @@ friend class CAddrManTest;
         LOCK(cs);
 
         Clear();
-        unsigned char nVersion;
-        s >> nVersion;
+
+        const auto stream_version_orig = s.GetVersion();
+
+        uint8_t format_tmp;
+        s >> format_tmp;
+        m_format = (Format)format_tmp;
```

Comment:
e025121c1ca4b86296c6b36386786522bd79a6e1
```suggestion
        m_format = static_cast<Format>(format_tmp);
```

Reply:
Done